### PR TITLE
Unify build call for .deb package + Dockerfile visual improvement

### PR DESCRIPTION
**Pull request by sepp0**
Changes:
1. `build_termux()` and `build_linux()` are merged into unified **`build_package()`**;
2. some minor visual improvements are added into Dockerfile.

**1. make-deb.sh**
I've noticed there is a number of almost identical `mkdir -p` calls (and others) in make-deb.sh script.
The only difference was the middle part of paths and platform name (TERMUX or DEBIAN).

To avoid that code duplication I've written a parametrized `build_package()`, which can be used for both Termux and Debian builds.

**2. Dockerfile**
Changes here are just for visual improvement.
In commit history I noticed that someone else has already done this, but somewhere in the midway Dockerfile got fully deleted and then reincarnated as it's "not so" efficient version.

So basically I'm just bringing back some of the original author's work.

Also I'm fixing my own error - in previous PR I forgot to add a newline at the EOF.

### CODE REVIEW

**Comment by sepp0** [comment]
@Yisus7u7 , I think you should take a look at this too :)

**Comment by Yisus7u7** [comment]
> @Yisus7u7 , I think you should take a look at this too :)

It looks great, I checked the code and it's correct.
